charts/README.md

```diff
@@ -32,13 +32,15 @@ See `values.yml` for more options.
 
 One option to highlight is the ability to enabled [Orbit] integration (which is
 disabled by default) and set a custom Orbit API endpoint (for development
-purposes). Use
+purposes). 
+
+To enable Orbit use:
```

Comment:
Agreed.
2 more things:
- There  is a typo on that line 33 (to enableD vs to enable), but that's outside the scope of this, maybe?
- I think saying somewhere in this document that Orbit is a 'management UI' is good for newcomers who are not yet familiar with the stack, but that's just an idea.

Reply:
Done

Reply:
I agree, let's not assume that readers know what Orbit is. Especially because the link to Orbit is to the signup/signin page which doesn't really explain what Orbit is or what it does. This is a better page to link to: https://www.zenko.io/admin/